models/user.py

```diff
@@ -42,7 +42,7 @@ def insert_roles():
         db.session.commit()
 
     def __repr__(self):
-        return '<Role \'%s\'>' % self.name
+        return f'<Role \'{self.name}\'>'
 
```

Comment:
**Problem:** you can use the super() function without any arguments to call the superclass's constructor without explicitly specifying the class name
**Suggestion:** `        return f'<Role \'{self.name}\'>'`